codec-rpm.cabal

```diff
@@ -39,15 +39,15 @@ library
                         attoparsec-binary >= 0.2 && < 0.3,
                         base >= 4.7 && < 5.0,
                         bytestring >= 0.10 && < 0.11,
-                        conduit >= 1.2.8 && < 1.3,
-                        conduit-combinators >= 1.1.0 && < 1.2,
-                        conduit-extra >= 1.1.15 && < 1.2,
+                        conduit >= 1.2.8 && < 1.4,
+                        conduit-extra >= 1.1.15 && < 1.4,
                         cpio-conduit >= 0.7.0 && < 0.8.0,
-                        lzma-conduit >= 1.1.3.3 && < 1.2,
+                        exceptions >= 0.9.0 && < 0.11.0,
```

Comment:
I think the lower end of "exceptions" can be dropped to 0.8.3, and that'll play nicer with Fedora 27